src/vs/workbench/contrib/chat/browser/codeBlockPart.ts

```diff
@@ -297,6 +373,23 @@ export class CodeBlockPart extends Disposable implements ICodeBlockPart {
 		}
 	}
 
+	protected override updateEditor(data: ISimpleCodeBlockData): void {
```

Comment:
From what I see, I'm assuming that the block won't update based on user changes in the source file? I think it makes the most sense to have it static- just checking.

Reply:
Currently it's a live just like normal editors. It's readonly but it will update if the file changes

Reply:
Ah, right. What is the use case where you would want it to update? I'm afraid that it might make the chat answer not make sense if you make big edits to the source?

Reply:
Mostly that it's more complicated to implement :)